README.md

```diff
@@ -0,0 +1,34 @@
+ORFik: R package for finding Open reading frames and creating ORF features
+==============================================================================
+
+This package is still under development, although this version is stable and can be used already.
+
+#### About
+Our R package’s scans for Open Reading Frames, giving GenomicRanges compatitable output.

```

Comment:
ORFik is an R package containing various functions for analysis of Ribo-Seq and RNA-Seq data. Currently it supports:

- Finding Open Reading Frames (very fast) in the genome of interest or on the set of transcripts.  
- Metaplots for Ribo-Seq allowing to spot the shift.  
- Shifting functions for the Ribo-Seq data.  
- Various measurements of gene identity eg. FLOSS, coverage, ORFscore, entropy.